### ISSUE TITLE
Connection: Staging Mode triggered when site can be accessed from 2 different addresses

### ISSUE THREAD

**Issue by lavacano** [Automattic/jetpack#22881]
Multiple instance webserver behind https ingress and load balancer "Your site is in Safe Mode because you have 2 Jetpack-powered sites that appear to be duplicates. 2 sites that are telling Jetpack they’re the same site. Learn more about safe mode."

### Impacted plugin

Jetpack

### Steps to Reproduce

1) Deploy wordpress on kubernetes or k8s https://cloud.google.com/wordpress
2) activate jetpack and link it to an account
3) 

Safe Mode has been activated
Your site is in Safe Mode because you have 2 Jetpack-powered sites that appear to be duplicates. 2 sites that are telling Jetpack they’re the same site. [Learn more about safe mode.](https://jetpack.com/redirect/?source=jetpack-support-safe-mode)

Please select an option
Move Jetpack data
Move all your settings, stats and subscribers to your other URL, 10.16.1.11. domain.com will be disconnected from Jetpack.

domain.com
10.16.1.11
Move your settings
or
Treat each site as independent sites
10.16.1.11 settings, stats, and subscribers will start fresh. domain.com will keep its data as is.

domain.com
10.16.1.11
Create a fresh connection
Or decide later and stay in Safe mode

### A clear and concise description of what you expected to happen.

I expect the jetpack plugin to share some sort of nonce in case multiple instances of a backend are used

### What actually happened

The jetpack plugin does not share some sort of nonce in case multiple instances of a backend are used

### Other information

wp-config.php url's must be set to the local ip address in order to export port 80 from eeach instance

### Operating System

Linux

### OS Version

COS

### Browser

Chrome/Chromium

### Browser Version(s)

24?

**Comment by jeherve** [comment]
> wp-config.php url's must be set to the local ip address

This does seem to be the source of the issue here; if your site is accessible by hitting the IP address directly, it does make it accessible at 2 different addresses (the domain and the IP). Would it be a way for you to redirect from the IP to the domain?

**Comment by lavacano** [comment]
yeah maybe i need to add it as a host k8s deployment for my business site is going to make my hair grey.  thanks.  will update.

**Comment by jeherve** [comment]
Closing this for now because of the lack of activity on this. We can always reopen in the future if needed.
